addons/account/static/src/components/autosave_many2many_tags/autosave_many2many_tags.js

```diff
@@ -2,9 +2,10 @@
 
 import { registry } from "@web/core/registry";
 import { useRecordObserver } from "@web/model/relational_model/utils";
-import { Many2ManyTagsField, many2ManyTagsField } from "@web/views/fields/many2many_tags/many2many_tags_field";
+import { many2ManyTagsField } from "@web/views/fields/many2many_tags/many2many_tags_field";
+import { Many2ManyTaxTagsField } from "../many2many_tax_tags/many2many_tax_tags";
 
-export class AutosaveMany2ManyTagsField extends Many2ManyTagsField {
+export class AutosaveMany2ManyTagsField extends Many2ManyTaxTagsField {
```

Comment:
If you do this the class name won't make sense anymore because it now also have the tax functionalities. You should rename both the class name and the widget to have "tax" inside it.

my suggestion:
new class name: AutosaveMany2ManyTaxTagsField
the const name below: autosaveMany2ManyTaxTagsField
the widget name: autosave_many2many_tax_tags

and update all occurrence of this widget name with that.
For now, (in master) all usage of the widget is only for tax_ids field, so it's fine to do this. But if there's any usage of the widget for fields that are not tax_ids in the future we'll have to refactor this again.